presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -57,7 +58,6 @@
 import static com.facebook.presto.common.type.DoubleType.DOUBLE;
 import static com.facebook.presto.common.type.IntegerType.INTEGER;
 import static com.facebook.presto.common.type.RealType.REAL;
-import static com.facebook.presto.common.type.RowType.Field;
```

Comment:
shall we keep this static import, and use Field in the code?

Reply:
done